Watchman.AwsResources.IntegrationTests/Service/CloudFront/CloudFrontSourceTests.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Amazon;
+using Amazon.CloudFront;
+using NUnit.Framework;
+using TestHelper;
+using Watchman.AwsResources.Services.CloudFront;
+
+namespace Watchman.AwsResources.IntegrationTests.Service.CloudFront
+{
+    public class CloudFrontSourceTests
+    {
+        [Test]
+        public async Task GetResourceNamesAsyncShouldReturnResults()

```

Comment:
I'm not sure of the value of this test, but they existed already, so I've added them here.

If the AWS Creds you test against don't have any Cloudfront distributions then it's going to fail

Reply:
yeah agreed, guess it's covered by other tests anyway?

Reply:
Yeah, it's covered elsewhere